packages/flutter/lib/src/gestures/pointer_router.dart

```diff
@@ -80,18 +80,17 @@ class PointerRouter {
       InformationCollector collector;
       assert(() {
         collector = () sync* {
-          yield DiagnosticsProperty<PointerEvent>('Event', event, style: DiagnosticsTreeStyle.errorProperty);
+          yield DiagnosticsProperty<PointerRouter>('PointerRouter', this, style: DiagnosticsTreeStyle.errorProperty);
```

Comment:
by convention property names should typically lower case. So perhaps
'router', 'route', and 'event' instead of PointerRouter, PointerRoute, and PointerEvent.